components/collective-page/CollectiveNotificationBar.js

```diff
@@ -45,6 +50,12 @@ const messages = defineMessages({
 });
 
 const getNotification = (intl, status, collective, host) => {
+  if (status === 'collectiveCreated' && host.name === 'Open Source Collective 501(c)(6)') {
```

Comment:
I think it's fragile to test the host here, especially the name.

Could we check if the collective is active or approved? `isActive` or `approvedAt` are the collective properties to look at.

Reply:
> I think it's fragile to test the host here, especially the name.
> 
> Could we check if the collective is active or approved? `isActive` or `approvedAt` are the collective properties to look at.

I agree with you, may be exact string comparison is not the right way, may be `isapproved` is a more generic way to check and can be adapted for other `createcollective` methods  too if it is there :)

Made apt changes